foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -185,5 +185,6 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   [The semantic difference between &lt;strong> and &lt;b> or &lt;em> and &lt;i> tags and when to use them.](https://medium.com/@zac_heisey/when-to-use-strong-b-em-and-i-tags-in-your-markup-fa4d0af8affb)
-*   [An interactive HTML text formatting article](https://www.w3schools.com/html/html_formatting.asp)
```

Comment:
Is there a particular reason for removing these?

Reply:
In the 'edit file' panel, this code is not missing. Would you like me to commit it anyways? Again my apologies, I don't want to mess things up further.